Eastern-Research-Group/mywaterway
# Feature/787 update dependencies

## Related Issues:
* [HMW-785](https://jira.epa.gov/browse/HMW-785)
* [HMW-787](https://jira.epa.gov/browse/HMW-787)

## Main Changes:
* Increased the contrast ratio for our info boxes. This one is for HMW-785, see commit 3e0c406 for more info. This was really the only item that I felt we needed to update code for. Most of the siteimprove items I flagged as false positives. The score won't improve until they do a manual review.
* Updated 3rd party dependencies.

## Steps To Test:
1. Navigate to http://localhost:3000/educators
2. Verify the message box looks ok and verify contrast ratio is at least 4.5 (the links was what siteimprove flagged)
3. Test as many other features as possible, primarily focussing on date sliders and arcgis

## TODO:
- [x] Run tests and code coverage
